.github/workflows/smoke-tests.yml

```diff
@@ -87,4 +87,18 @@ jobs:
           exit 1
         fi
 
-
+    - name: Port Forward frontend
+      run: |  
+        set -x
+        port_forward() {
+            # get name of frontend pod
+            local frontend_pod=$(kubectl get pods -l=app=frontend -o=jsonpath='{.items[0].metadata.name}')
+            kubectl port-forward $frontend_pod 3000:8080   
+        }
+        port_forward &
```

Comment:
Do we know if this process is cleaned up after the CI run?

Reply:
On my fork the Complete Job logs
```
Cleaning up orphan processes
Terminate orphan process: pid (15260) (bash)
Terminate orphan process: pid (15272) (kubectl)
```
And I've been able to verify on my fork's runner that it's no longer running.  Will verify on this repo's runner.